render/interfaceFeatures.go

```diff
@@ -0,0 +1,18 @@
+package render
```

Comment:
Should this be in render? Or should there be something analogous to the alg package as this might be re-used down the line outside of render.